#image topic to subscribe, confidence for object detection, non-maximum suppression
sensor_msgs/Image image_raw
string dataset
float32 confidence
float32 nms
---
#bounding box image for testing, name and coords of each object detected
Detection[] detected_objects
